app.js

```diff
@@ -23,17 +24,51 @@ let draws = 0;
 
 startButton.addEventListener('click', () => { 
 
+    
     const currentComputerPick = randomThrow();
+    console.log(currentComputerPick);
     const checkedRadioButton = document.querySelector(':checked');
     const currentUserPick = checkedRadioButton.value;
- 
+    console.log(currentUserPick);
+    
+    const gameOutcome = doesUserWin(currentUserPick, currentComputerPick);
+    console.log(gameOutcome);
+    // doesUserWin(currentUserPick, currentComputerPick);
+  
     // ------------------------------------------------------------------------------------
     // set event listeners to update state and DOM
     // If picks are the same they tie. 
  
-    doesUserWin(currentUserPick, currentComputerPick)
+    if (gameOutcome === 'Tie'){
+        draws++;
+        drawsConverted.textContent = draws;
+        gameResult.textContent = 'You tied!';
+    }
+    
+    if (gameOutcome === 'You Win'){
+            wins++;
+            winsConverted.textContent = wins;
+            gameResult.textContent = 'You won!';
+        }
+    
+    if (gameOutcome === 'You Lose'){
+                losses++;
+                lossesConverted.textContent = losses;
+                gameResult.textContent = 'You lost!';
+        }
+    })
+
+resetButton.addEventListener('click', () => {
+    let wins = 0;
```

Comment:
You don't need to use 'let' here you've already created the variable on line 17-19 in app.js. I think the let causes a little bug where the state it's not setting properly.